src/mappings/entities/mapMultiSig.ts

```diff
@@ -158,6 +159,21 @@ const handleMultiSigProposalAdded = async (
   const multisigId = getTextValue(rawMultiSigAddress);
   const proposalId = getNumberValue(rawProposalId);
   const creatorAccount = extrinsic?.extrinsic.signer.toString();
+  const parsed = (extrinsic?.extrinsic?.toHuman() as any).method.args || [];
+
+  const module = parsed[1]?.value.call_module;
+  const call = parsed[1]?.value.call_function;
```

Comment:
@polymath-eric do you think we should parse these into our enum types (CallIdEnum and ModuleIdEnum)? That way it would be easily filterable.

Reply:
For filterability wouldn't they need to go onto the `MultiSigProposal` itself. 

I thought JSON fields get worse indexing compared to regular columns.

In general I agree it would be better to use the enum value though.